browser/src/core/Socket.js

```diff
@@ -1252,6 +1255,11 @@ app.definitions.Socket = L.Class.extend({
 		}
 	},
 
+	_exportAsCallback: function(command) {
+		this._map.hideBusy();
+		this._map.uiManager.showInfoModal('exported-success', _('Exported to storage'), _('Successfully exported: ') + decodeURIComponent(command.filename), '', _('OK'));
```

Comment:
here is `export` result handling, while below we see standard `save as` - closing the document and opening the new one